src/main/java/com/aws/greengrass/util/platforms/windows/WindowsPlatform.java

```diff
@@ -409,17 +404,11 @@ public CmdDecorator() {
 
         @Override
         public ShellDecorator withShell(String shell) {
-            switch (shell) {
```

Comment:
why can't I specify withShell("cmd")? Don't break that. If we're going to have overrides like I think we will, then customers could specify either "cmd" or "powershell" explicitly.

Reply:
Removed because cmd is the default. I think it makes sense that user may want to explicitly indicate "cmd" in the code. Reverted.